students/Chris_Hallen/lesson02/fizzBuzz.py

```diff
@@ -0,0 +1,14 @@
+def fizzBuzz():
```

Comment:
per pep8 style guide function names (and variables) should be snake_case -> `fizz_buzz`